README.md

```diff
@@ -70,17 +97,20 @@ For this setup to work you need to have a Docker service installed. You can get
 Create `TimeSeries` object from a Pandas DataFrame, and split in train/validation series:
 
 ```python
+import pandas as pd
 from darts import TimeSeries
 
 df = pd.read_csv('AirPassengers.csv', delimiter=",")
 series = TimeSeries.from_dataframe(df, 'Month', '#Passengers')
```

Comment:
Since just a few moments ago this needs to be `TimeSeries.from_dataframe(df, 'Month', ['#Passengers'])` due to newly added multivariate support.

Reply:
I added back support of single string type for the value_cols argument, so you can change it back to how it was now. Sorry for the trouble!